src/riak_kv_pb_object.erl

```diff
@@ -190,8 +193,8 @@ process(#rpbputreq{bucket=B0, type=T, key=K, vclock=PbVC,
     case Result of
         consistent ->
             process(Req#rpbputreq{if_not_modified=undefined,
-                                  if_none_match=consistent},
-                    State);
+                                  if_none_match=undefined},
```

Comment:
It seems weird to set `if_none_match` to `undefined` here, but then later set it to `true` for the `put` options.